README.md

```diff
@@ -512,7 +513,7 @@ Other Modules, IDEs, Resources, Cool Stuff, ...
 * [Stanford Online](http://online.stanford.edu/courses)
 * [Coursera](https://www.coursera.org/)
 * [Programming Books Sandbox](https://github.com/bharathwaaj/sandbox/tree/master/Programming%20books)
-* [Awesome Courses](https://github.com/prakhar1989/awesome-courses)
+* [550+ Free Online Programming &amp; Computer Science Courses You Can Start This October](https://www.freecodecamp.org/news/550-free-online-programming-computer-science-courses-you-can-start-this-october/)
```

Comment:
Replaced `Awesome Courses` link because it's a duplicate of the one posted under the `Top Self-learning Resources` section